templates/workloads/services/lb-web/manifest.yml

```diff
@@ -13,6 +13,10 @@ http:
   path: '{{.Path}}'
   # You can specify a custom health check path. The default is "/".
   # healthcheck: '{{.HealthCheck.HealthCheckPath}}'
+{{- if .AppDomain}}
+  # You can specify a custom domain alias. The default domain name is '{{.Name}}.${EnvName}.${AppName}.{{.AppDomain}}'.
+  # alias: 'other.{{.AppDomain}}'
```

Comment:
It just occurred to me that we shouldn't write `alias` as a field yet. We're most likely going to do a release before this feature is shipped so if customers see this field then they'll think it's supported.

Should we remove `DomainAlias       *string                 yaml:"alias"` from the manifest pkg for now and just have it supported in the `template` pkg so that you're unblocked with the remaining milestones?